macs/README.md

```diff
@@ -31,8 +31,11 @@ These are maintained by the build infra team.
 - Login to user hetzner with the given password
 - Set up SSH keys in the hetzner user
 - Elevate with `sudo su`
-- Install latest system updates
-  - softwareupdate --install --all --restart
+- ~~Install latest system updates~~
+  - ~~softwareupdate --install --all --restart~~
+- Disable auto-updates:
+  - We are currently seeing performance regression in macOS Sequoia.
+  - So to not have the machines auto-upgrade, we use: `sudo softwareupdate --schedule off`
```

Comment:
As of https://github.com/LnL7/nix-darwin/pull/992, the option [system.defaults.SoftwareUpdate.AutomaticallyInstallMacOSUpdates](https://daiderd.com/nix-darwin/manual/index.html#opt-system.defaults.SoftwareUpdate.AutomaticallyInstallMacOSUpdates) should work